code/lib/cli/src/sandbox-templates.ts

```diff
@@ -139,7 +139,7 @@ const baseTemplates = {
     },
   },
   'react-webpack/18-ts': {
-    name: 'React 18 Webpack5 (TS)',
+    name: 'React Webpack (TypeScript)',
```

Comment:
What do you think is clearer for the general audience?

1:
<img width="371" alt="image" src="https://user-images.githubusercontent.com/1671563/237052424-01e8975e-dcf6-4450-ac5d-0279921921d7.png">

2:
<img width="367" alt="image" src="https://user-images.githubusercontent.com/1671563/237052574-dbcc1df6-2101-4e45-97fb-3e0fd895ce40.png">

Reply:
The first one. If we don't think that's the case, then we should have framework versions on all of the sandboxes. Which would be okay to me, except it wouldn't technically be true, since we're just using `@latest`, so we'd have to keep the string up-to-date.

Reply:
The only reason I opted for `v3` in Vue, was because I get a sense that the Vue community still hasn't accepted v3 as being the "latest", that v2 and v3 are forks of each other. but I could be wrong.

Reply:
Maybe we:
- change JavaScript to JS
- add (latest) for places that do not use v